### PR TITLE
NMS-13640: Remove SQS support

### PR DESCRIPTION
**Pull request by fooker**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13640

### CODE REVIEW

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 1]
```diff
@@ -29,9 +29,9 @@ If you wish to revert to the previous behavior, create an `$OPENNMS_HOME/etc/ope
 RUNAS=root
 ----
 
-=== SQS Minion RPC Support Dropped
+=== SQS Minion IPC Support Dropped
 
-Support for Amazon SQS for Minion RPC has been removed.
+Support for Amazon SQS for Minion Communication has been removed.
```
Comment (Bonrob2) [inline docs/modules/releasenotes/pages/whatsnew.adoc:34]:
```suggestion
Support for Amazon SQS for Minion communication has been removed.
```

**Comment by cgorantla** [comment]
Yay!!  Looks good to me!

**Review by cgorantla — APPROVED** [review]

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Just one change in doc files. Looks good.

**Review by Bonrob2 — APPROVED** [review]
Doc files LGTM!

**Pull request merged**